tools/aws-ip.sh

```diff
@@ -1,3 +1,8 @@
 ipv4=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
 
-echo '\n\nAPI_URL=http://'$ipv4':3000' >> ../telescope/.env
+cp config/env.development config/env.remote
+
+sed -i "s/for Development/for Remote Development/g" config/env.remote
+sed -i "s/localhost/$ipv4/g" config/env.remote
+
+cp config/env.remote .env
```

Comment:
Nice fix, way simpler!  Let's `cp` the file once to `.env` and run `sed` on it there.